exercises/08-exercise.py

```diff
@@ -0,0 +1,132 @@
+'''
+Author: Jorge Diaz
+Exercise 8
+'''
+import uuid
+import pandas as pd
+from databroker.tests.utils import temp_config
+from databroker import Broker
+from databroker.assets.handlers_base import HandlerBase
+
+# this will create a temporary databroker object with nothing in it
+db = Broker.from_config(temp_config())
+
+fc = 7.62939453125e-05
+adc2counts = lambda x: ((int(x, 16) >> 8) - 0x40000) * fc \
+        if (int(x, 16) >> 8) > 0x1FFFF else (int(x, 16) >> 8)*fc
+
+
+class MyHandler(HandlerBase):
```

Comment:
Can you call this `PizzaBoxAnHandler`?

Can you make a second handler that reads 'en' files called `PizzaBoxEnHandler`?

The conversions are:
```py3
fc = 7.62939453125e-05
adc2counts = lambda x: ((int(x, 16) >> 8) - 0x40000) * fc \
        if (int(x, 16) >> 8) > 0x1FFFF else (int(x, 16) >> 8)*fc
enc2counts = lambda x: int(x) if int(x) <= 0 else -(int(x) ^ 0xffffff - 1)
```

Can you:
-  modify the An handler to return only two columns: 'timestamp', and 'adc' where 'adc' is the converted adc from counts?
- modify the En handler to return ['timestamp', 'encoder'] where 'encoder' is the before last column in the file converted using `enc2counts`?